packages/mineral-ui-tokens/tokens/aliases.json

```diff
@@ -7,10 +7,15 @@
     "fontSize_mouse": "0.6875rem",
     "fontSize_prose": "1rem",
     "fontSize_ui":    "0.875rem",
+    "fontSize_ui_sm": "0.75rem",
 
     "fontWeight_regular":   400,
     "fontWeight_semiBold":  600,
     "fontWeight_bold":      700,
-    "fontWeight_extraBold": 800
+    "fontWeight_extraBold": 800,
+
+    "borderWidth_md": 1,
```

Comment:
I would say 1px is actually "small" `sm` not medium. since you can't technically get smaller than a 1px border.
make this:
borderWidth_sm: 1
borderWidth_md: 1.5
borderWidth_lg: 2

Reply:
Agree